lib/server.js

```diff
@@ -313,7 +313,7 @@ function Server (options) {
     return c
   }
 
-  function newConnection (conn) {
+  self.newConnection = function (conn) {
     setupConnection(conn)
```

Comment:
```suggestion
    // TODO: make `newConnection` available on the `Server` prototype
    // https://github.com/ldapjs/node-ldapjs/pull/727/files#r636572294
    setupConnection(conn)
```

We'll solve this at a later date.